telluric/georaster.py

```diff
@@ -1630,10 +1554,17 @@ def get_tile(self, x_tile, y_tile, zoom,
         :param blocksize: tile size  (x & y) default 256, for full resolution pass None
         :return: GeoRaster2 of tile
         """
-        coordinates = mercantile.xy_bounds(x_tile, y_tile, zoom)
-        window = self._window(coordinates)
-        return self.get_window(window, bands=bands,
-                               xsize=blocksize, ysize=blocksize)
+        roi = GeoVector.from_xyz(x_tile, y_tile, zoom)
+        coordinates = roi.get_bounds(WEB_MERCATOR_CRS)
+        window = self._window(coordinates, to_round=False)
+        window.col_off = round(window.col_off)
```

Comment:
Aren't these four lines essentially `window.round_shape(op="round")`, which isn't supported? Perhaps we should open an issue on rasterio.

Reply:
I guess you are right, 
currently they support only "floor" or "ceil"